Libraries/Image/RCTImageUtils.m

```diff
@@ -314,7 +314,7 @@ BOOL RCTUpscalingRequired(CGSize sourceSize, CGFloat sourceScale,
   return (__bridge_transfer id)imageProperties;
 }
 
-NSData *__nullable RCTGetImageData(CGImageRef image, float quality)
+NSData *__nullable RCTGetImageData(CGImageRef image, float quality, NSInteger orientation)
 {
   NSDictionary *properties;
```

Comment:
If we changing this quite radically (eg switching to using `NSMutableDictionary`), we can rename it to `imageOptions` (actual correct name from `CGImageDestinationCreateWithData` signature).